src/encoded/upgrade/award.py

```diff
@@ -31,3 +31,15 @@ def award_5_6(value, system):
     # https://encodedcc.atlassian.net/browse/ENCD-4682
     if 'title' not in value:
         value['title'] = value['name']
+
+
+@upgrade_step('award', '6', '7')
+def award_6_7(value, system):
+    # https://encodedcc.atlassian.net/browse/ENCD-4711
+    for m in value['milestones']:
+        if re.match('single-nuclei ATAC-seq', m):
+            new_milestone = m.replace('single-nuclei', 'single-nucleus')
```

Comment:
`m` here, I think, would be a dict rather than a string. May need some changes here... What's interesting is we don't have any awards with milestones. In principle, however, we still need to make this right...

Reply:
Corrected, I am now handling each milestone as a dict.